docs/command/atlas-api-accessTracking-listAccessLogsByClusterName.txt

```diff
@@ -12,9 +12,9 @@ atlas api accessTracking listAccessLogsByClusterName
    :depth: 1
    :class: singlecol
 
-`Public Preview: please provide feedback <https://feedback.mongodb.com/forums/930808-atlas-cli>`_: Returns the access logs of one cluster identified by the cluster's name.
+Returns the access logs of one cluster identified by the cluster's name.
 
-The atlas api sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API and is currently in Public Preview (please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli).
+The atlas api sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API.
```

Comment:
Minor: I know this is the current phrasing but it's always seemed a little odd to me. Suggested changes in *bold*:

>  "The atlas api sub-command *is* automatically generated from the MongoDB Atlas Admin API *and* offers full coverage of the Admin API"